From 918ec016734835b741230acc446393e90924af81 Mon Sep 17 00:00:00 2001
From: Paul Lesniewski <paul@squirrelmail.org>
Date: Tue, 16 Nov 2010 19:53:11 +0000
Subject: [PATCH 10/50] Synchronize connection descriptor struct's username
 size with the username buffer.  Thanks to Daniel Marczisovszky for the catch.

git-svn-id: https://svn.code.sf.net/p/squirrelmail/code/trunk/imap_proxy@14074 7612ce4b-ef26-0410-bec9-ea0150e637f0
---
 include/imapproxy.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/imapproxy.h b/include/imapproxy.h
index e58a5af..dd30a38 100644
--- a/include/imapproxy.h
+++ b/include/imapproxy.h
@@ -257,7 +257,7 @@ struct IMAPTransactionDescriptor
 struct IMAPConnectionContext
 {
     struct IMAPConnectionDescriptor *server_conn;
-    char username[64];                  /* username connected on this sd     */
+    char username[MAXUSERNAMELEN];      /* username connected on this sd     */
     char hashedpw[16];                  /* md5 hash copy of password         */
     time_t logouttime;                  /* time the user logged out last     */
     struct IMAPConnectionContext *next; /* linked list next pointer          */
-- 
2.7.4

